src/Control/Session.php

```diff
@@ -372,7 +373,7 @@ private function buildCookieParams(HTTPRequest $request): array
             }
         }
 
-        $sameSite = static::config()->get('cookie_samesite') ?? Cookie::SAMESITE_LAX;
+        $sameSite = static::config()->get('cookie_samesite') ?? Cookie::SAMESITE_STRICT;
```

Comment:
This and any other changes from lax to strict in this class are necessary as part of https://github.com/silverstripe/silverstripe-framework/issues/11597

Reply:
It's only used in one place so that seems excessive - but done anyway. Private method so we can just axe it later if we want.